source/Calamari.Tests/Fixtures/Certificates/ImportCertificateCommandFixture.cs

```diff
@@ -53,12 +54,15 @@ public void AddingCertToStore_AddsCert()
             cert.EnsureCertificateNotInStore(storeName, certificateStoreLocation);
         }
 
+#if WINDOWS_CERTIFICATE_STORE_SUPPORT
```

Comment:
This test still relies on some of the capabilities not yet available to netcore (see above), it will be exposed again in a later PR